gravity-forms/gw-choice-counter.php

```diff
@@ -90,7 +90,9 @@ function output_script() {
 					// Event handler for all listeners to avoid DRY and to maintain a pointer reference to the function
 					// which we can use to explicity unbind event handlers
 					self.updateChoiceEventHandler = function() {
-						self.updateChoiceCount( self.formId, self.choiceFieldIds, self.countFieldId, self.values );
+						setTimeout( function() {
```

Comment:
What about the following?

I think it might be slightly more reliable as I believe it will ensure that the code runs after the DOM changes but before the next repaint

```suggestion
						requestAnimationFrame(function() {
							self.updateChoiceCount( self.formId, self.choiceFieldIds, self.countFieldId, self.values );
						});
```